From 729db839b762a472444bacff22a1cb8870635272 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Wed, 17 Jul 2013 14:53:49 +0400
Subject: [PATCH 10/10] Install into BINDIR instead of SBINDIR

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 src/Makefile.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/Makefile.in b/src/Makefile.in
index 1a1fa41..1578183 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -92,7 +92,7 @@ DESTDIR =
 ETCDIR = $(DESTDIR)@sysconfdir@/ejabberd
 
 # /sbin/
-SBINDIR = $(DESTDIR)@sbindir@
+BINDIR = $(DESTDIR)@bindir@
 
 # /lib/ejabberd/
 EJABBERDDIR = $(DESTDIR)@libdir@/ejabberd
@@ -201,11 +201,11 @@ install: all
 	install -b -m 644 $(G_USER) inetrc $(ETCDIR)/inetrc
 	#
 	# Administration script
-	[ -d $(SBINDIR) ] || install -d -m 755 $(SBINDIR)
-	install -m 755 $(G_USER) ejabberdctl.example $(SBINDIR)/ejabberdctl
+	[ -d $(BINDIR) ] || install -d -m 755 $(BINDIR)
+	install -m 755 $(G_USER) ejabberdctl.example $(BINDIR)/ejabberdctl
 	#
 	# Init script
-	sed -e "s*@ctlscriptpath@*$(SBINDIR)*" \
+	sed -e "s*@ctlscriptpath@*$(BINDIR)*" \
 		-e "s*@installuser@*$(INIT_USER)*" ejabberd.init.template \
 		> ejabberd.init
 	chmod 755 ejabberd.init
@@ -273,7 +273,7 @@ install: all
 uninstall: uninstall-binary
 
 uninstall-binary:
-	rm -f  $(SBINDIR)/ejabberdctl
+	rm -f  $(BINDIR)/ejabberdctl
 	rm -fr $(DOCDIR)
 	rm -f  $(BEAMDIR)/*.beam
 	rm -f  $(BEAMDIR)/*.app
-- 
1.8.3.1

